README.md

```diff
@@ -31,29 +32,37 @@ The `master` branch is regularly built and tested, but is not guaranteed to be
 completely stable. [Tags](https://github.com/bitcoin/bitcoin/tags) are created
 regularly to indicate new official, stable release versions of Bitcoin Core.
 
-The contribution workflow is described in [CONTRIBUTING.md](CONTRIBUTING.md)
-and useful hints for developers can be found in [doc/developer-notes.md](doc/developer-notes.md).
+The contribution workflow is described in [CONTRIBUTING.md](CONTRIBUTING.md).
+
+Useful hints for developers can be found in [doc/developer-notes.md](doc/developer-notes.md).
+
+Instructions for building Bitcoin Core from source are in [doc/build-unix.md](https://github.com/bitcoin/bitcoin/blob/improvements/readme/doc/build-unix.md#to-build).
+See `doc/build-*.md` for other platforms.
 
 Testing
 -------
 
-Testing and code review is the bottleneck for development; we get more pull
-requests than we can review and test on short notice. Please be patient and help out by testing
-other people's pull requests, and remember this is a security-critical project where any mistake might cost people
-lots of money.
+Testing and code review is the bottleneck for development; we get more pull requests
+than we can review and test on short notice. Please be patient and help by testing
+other people's pull requests.
+
+Remember that this is a security-critical project. Mistakes can cost people lots of money.
 
 ### Automated Testing
 
 Developers are strongly encouraged to write [unit tests](src/test/README.md) for new code, and to
-submit new unit tests for old code. Unit tests can be compiled and run
-(assuming they weren't disabled in configure) with: `make check`. Further details on running
-and extending unit tests can be found in [/src/test/README.md](/src/test/README.md).
+submit new unit tests for old code. Unit tests can be compiled and run with `make check`,
+assuming they weren't disabled in `configure`. Further details on running, writing,
+and extending unit tests can be found in [src/test/README.md](src/test/README.md).
+
+There are also [regression and integration tests](/test) written in Python 3. These tests
+can be run with `test/functional/test_runner.py` if the [test dependencies](/test) are installed.
 
-There are also [regression and integration tests](/test), written
-in Python, that are run automatically on the build server.
-These tests can be run (if the [test dependencies](/test) are installed) with: `test/functional/test_runner.py`
+Travis CI and AppVeyor CI make sure that every pull request is built for Linux, Windows,
+and macOS. Unit, integration, regression, and sanity tests are run automatically.
 
-The Travis CI system makes sure that every pull request is built for Windows, Linux, and macOS, and that unit/sanity tests are run automatically.
+See https://travis-ci.org/bitcoin/bitcoin and https://ci.appveyor.com/project/DrahtBot/bitcoin
+to view output from the Travis and AppVeyor build servers.
```

Comment:
They are already linked from commits which were run and not linked for commits which are not run. No need to put them into the readme, imo.